docs-ref-conceptual/install-azure-cli-linux.md

```diff
@@ -16,8 +16,8 @@ keywords: linux cli, azure cli linux, install azure cli ubuntu, install azure cl
 
 # Install the Azure CLI on Linux
 
-This article explains how to install the Azure CLI on Linux. Select the appropriate package manager for your distribution from the options above.
-Using a Linux distribution's package manager is recommended, but you may manually install the Azure CLI on Linux by selecting the [Install script](?pivots=script) option.
+The Azure CLI is a cross-platform command-line tool that can be installed locally on Linux computers. You can use the Azure CLI on Linux to connect to Azure and execute administrative commands on Azure resources. The CLI on Linux allows the execution of various commands through a terminal using interactive command-line prompts or a script.
```

Comment:
@daphnemamsft, This sentence was shortened on purpose.  Please rewrite without adding content.

Reply:
This article explains how to install the Azure CLI on Linux. Select the appropriate package manager for your distribution from the options at the top of the page.
Using a Linux distribution's package manager is recommended, but you may manually install the Azure CLI on Linux by selecting the [Install script](?pivots=script) option.